worker-api/src/main/java/com/hpe/caf/api/worker/QueueTaskMessage.java

```diff
@@ -1,261 +0,0 @@
-/*
- * Copyright 2015-2023 Open Text.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.hpe.caf.api.worker;
-
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
-
-import jakarta.validation.constraints.NotNull;
-
-/**
- * The generic task message class to be serialised from or to a queue. This will contain the serialised worker-specific data inside.
- */
-public final class QueueTaskMessage
-{
-    public static final int TASK_MESSAGE_VERSION = 4;
```

Comment:
Dropping support for the V4 format message is technically a breaking change so we should note it in the release notes.